### PR TITLE
docs: document per-client access token strategy

### PR DESCRIPTION
**Pull request by hperl**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by vinckr — APPROVED** [review]
lgtm! 

is this already merged in hydra/on prod?

File: docs/oauth2-oidc/jwt-access-token.mdx [hunk 1]
```diff
@@ -46,13 +46,30 @@ access token has expired. Refresh tokens are always opaque tokens in Ory because
...
+### Per-client configuration
+
+To configure Ory OAuth2 to issue JWT access tokens for a specific client, you'll need to update the client configuration as Ory
+defaults to opaque tokens:
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock'
+import setAccessTokenStrategyExample from '!!raw-loader!../../code-examples/sdk/typescript/src/oauth2/client-set-access-token-strategy.ts'
+
+<CodeBlock language="tsx">{setAccessTokenStrategyExample}</CodeBlock>
```
Comment (Benehiko) [inline docs/oauth2-oidc/jwt-access-token.mdx:68]: Couldn't this also be done through cURL and Go? Maybe we should add it as well?
https://github.com/ory/docs/tree/master/code-examples/curl
https://github.com/ory/docs/tree/master/code-examples/sdk/go

**Review by Benehiko — CHANGES_REQUESTED** [review]
Just a small comment

File: docs/oauth2-oidc/jwt-access-token.mdx (hunk 1 above)
Comment (kmherrmann) [reply docs/oauth2-oidc/jwt-access-token.mdx:68]: Let's do this in a follow-up PR to have something shipped right now

**Review by Benehiko — APPROVED** [review]

**Pull request merged**